spec/rails_helper.rb

```diff
@@ -46,6 +47,12 @@
     Dictionary::Quant.instance.reset
     Dictionary::Qual.instance.reset
     Dictionary::Ind.instance.reset
+
```

Comment:
this feels out of place in this PR. @tsubik @agnessa ?

Reply:
@tiagojsag This is cherry-picked from @agnessa branch to stop unrelated tests erroring on CI after merging in develop branch.